src/main/java/org/gridsuite/loadflow/server/LoadFlowService.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.loadflow.server;
+
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.loadflow.LoadFlow;
+import com.powsybl.loadflow.LoadFlowParameters;
+import com.powsybl.loadflow.LoadFlowResult;
+import com.powsybl.network.store.client.NetworkStoreService;
+import com.powsybl.openloadflow.OpenLoadFlowParameters;
+import com.powsybl.openloadflow.network.MostMeshedSlackBusSelector;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.UUID;
+
+/**
+ * @author Franck Lecuyer <franck.lecuyer at rte-france.com>
+ */
+@ComponentScan(basePackageClasses = {NetworkStoreService.class})
+@Service
+class LoadFlowService {
+
+    @Autowired
+    private NetworkStoreService networkStoreService;
+
+    private Network getNetwork(UUID networkUuid) {
+        try {
+            return networkStoreService.getNetwork(networkUuid);
+        } catch (PowsyblException e) {
+            throw new ResponseStatusException(HttpStatus.NOT_FOUND, "Network '" + networkUuid + "' not found");
+        }
+    }
+
+    void loadFlow(UUID networkUuid) {
+        Network network = getNetwork(networkUuid);
+
+        // launch the load flow on the network
+        LoadFlowParameters parameters = new LoadFlowParameters();
```

Comment:
I guess it is because with you test case it is the only way to have it running?
You can keep it as default but only if no parameters are passes to the query

Reply:
done